client/contracts/truffle/contracts/oracle/OracleMgr.sol

```diff
@@ -31,19 +31,17 @@ contract OracleMgr is Pausable, Initializable {
     
     mapping (address => Oracle) private oracleRegistry;
     
-    // oraclePool contains the oracle identity ordered by the biggest deposit to
-    // the smallest deposit.
     address[] private oraclePool;
 
     OraclePrice[] private prices;
 
-    modifier onlyOracle {
-        require(isOracle(msg.sender));
+    modifier onlyOracle(address oracle) {
```

Comment:
Why change that?

Reply:
because the it's no longer the msg sender that registers himself. It's kowala